bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/component.jsx

```diff
@@ -151,25 +153,48 @@ class PresentationUploader extends Component {
   }
 
   handleConfirm() {
-    const { presentations } = this.state;
+    const presentationsToSave = this.state.presentations
+      .filter(p => !p.upload.error && !p.conversion.error);
 
     this.setState({
       disableActions: true,
       preventClosing: true,
+      presentations: presentationsToSave,
     });
 
-    return this.props.handleSave(presentations)
+    return this.props.handleSave(presentationsToSave)
       .then(() => {
+        const { presentations, oldCurrentId } = this.state;
+
+        const hasError = presentations.some(p => p.upload.error || p.conversion.error);
+
+        if (!hasError) {
+          this.setState({
+            disableActions: false,
+            preventClosing: false,
+          });
+
+          return;
+        }
+
+        // if theres error we dont want to close the modal
         this.setState({
           disableActions: false,
-          preventClosing: false,
+          preventClosing: true,
+        }, () => {
+          // if the selected current has error we revert back to the old one
+          const newCurrent = presentations.find(p => p.isCurrent);
+          if (newCurrent.upload.error || newCurrent.conversion.error) {
+            this.handleCurrentChange(oldCurrentId);
+          }
         });
       })
       .catch((error) => {
+        console.error(error);
```

Comment:
could we log this on the sever too/instead?

Reply:
By calling console.error this is already being logged to the server.